classes/functions.php

```diff
@@ -154,11 +154,13 @@ public function issuesList($series_id) {
 				$this->comic_id = $row ['comic_id'];
 				$this->wiki_id = $row ['wiki_id'];
 				$this->issue_number = $row ['issue_number'];
-				$this->story_name = $row ['story_name'];
-				$this->issue_list .= "<tr>\n";
-				$this->issue_list .= "<td class=\"mdl-data-table__cell--non-numeric\">" . "<a href=\"comic.php?comic_id=$this->comic_id\">" . $this->issue_number . "</a>" . "</td>";
-				$this->issue_list .= "<td class=\"mdl-data-table__cell--non-numeric\">" . "<a href=\"comic.php?comic_id=$this->comic_id\">" . $this->story_name . "</a>" . "</td>";
-				$this->issue_list .= "</tr>";
+				$this->release_date = DateTime::createFromFormat('Y-m-d', $row ['release_date'])->format('M Y');
```

Comment:
This long string is to convert the database standard YYYY-MM-DD format into the comic book style of Month YYYY (Jan 1994) as seen on the Issue index.